src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/globals.groovy

```diff
@@ -26,8 +26,10 @@ l.layout(title:_("Pipeline Syntax: Global Variable Reference"), norefresh: true)
           h2(_("Variables"))
           dl(class:'steps variables root'){
             for (GlobalVariable v : snippetizer.getGlobalVariables()) {
-              dt {
-                code(v.getName())
+              dt(id: v.getName()) {
+                a(href: '#' + v.getName()) {
```

Comment:
Huh? This will create a link to a nonexistent anchor.

I think what you meant was

```groovy
dt {
  a(name: v.name) {
    code(v.name)
  }
}
```

Did you test this in a browser?